manifests/vhost.pp

```diff
@@ -2312,6 +2313,11 @@
       require => Concat["${priority_real}${filename}.conf"],
       notify  => Class['apache::service'],
     }
+  } elsif !$enable and $apache::vhost_enable_dir {
+    file { "${priority_real}${filename}.conf symlink":
+      ensure => 'absent',
+      path   => "${vhost_enable_dir}/${priority_real}${filename}.conf",
+      notify  => Class['apache::service'],
```

Comment:
The `file` block seems to be missing a closing `}`.  
I suppose this is a good indicator that this should be tested before merging.